From: "K. Y. Srinivasan" <kys@microsoft.com>
Date: Wed, 18 Mar 2015 12:29:29 -0700
Patch-mainline: v4.1-rc1
Subject: Drivers: hv: vmbus: Export the vmbus_sendpacket_pagebuffer_ctl()
Git-commit: b3a19b36ad7450d4c8caeb7d9f623af3da32a578
References: fate#317533

Export the vmbus_sendpacket_pagebuffer_ctl() interface. This export will be
used by the netvsc driver.

Signed-off-by: K. Y. Srinivasan <kys@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Acked-by: <ohering@suse.de>
---
 drivers/hv/channel.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/hv/channel.c b/drivers/hv/channel.c
index 2c8206d..fddd3b5 100644
--- a/drivers/hv/channel.c
+++ b/drivers/hv/channel.c
@@ -707,6 +707,7 @@ int vmbus_sendpacket_pagebuffer_ctl(struct vmbus_channel *channel,
 
 	return ret;
 }
+EXPORT_SYMBOL_GPL(vmbus_sendpacket_pagebuffer_ctl);
 
 /*
  * vmbus_sendpacket_pagebuffer - Send a range of single-page buffer
